drivers/usb/device/Kconfig

```diff
@@ -50,8 +50,7 @@ config USB_DC_STM32
 	select USE_STM32_HAL_PCD
 	select USE_STM32_HAL_PCD_EX
 	help
-	  Enable USB support on the STM32 F0, F1, F2, F3, F4, F7, L0, L4, G4, U5 family of
-	  processors.
+	  Enable USB support on the STM32 family of processors.
```

Comment:
Since you are touching it, please change it to

"Enable STM32 family USB **device controller shim driver**."